R/shapley_unsampl.R

```diff
@@ -1,7 +1,20 @@
-#######New version number 5
-#some style correction
-#deleted previous versions
-
+.onAttach <- function(libname, pkgname) {
+  packageStartupMessage("Welcome to the ShapleyR package!")
+}
+#' Calculates the exact shapley value for every player.
```

Comment:
Change it to something like: " Calculates the exact shapley value (https://en.wikipedia.org/wiki/Shapley_value) for every player based on the algorithm for cooperative game theory."